app/src/main/java/by/naxa/soundrecorder/services/RecordingService.java

```diff
@@ -152,8 +152,12 @@ public void onDestroy() {
     public void setFileNameAndPath(boolean isFilePathTemp) {
         if (isFilePathTemp) {
             mFileName = getString(R.string.default_file_name) + (++tempFileCount) + "_" + ".tmp";
+            File file = new File(getExternalCacheDir() + Paths.SOUND_RECORDER_FOLDER);
```

Comment:
minor: prefer to use `Paths.combine` when you need to concatenate path components.

Reply:
changes made. please check @naXa777